File: sublib/src/main/java/com/enhtmv/sublib/common/http/SubProxy.java
package com.enhtmv.sublib.common.http;

public class SubProxy {

    private String host;
    private String username;
    private String password;
    private int port;

    public SubProxy(String host, String username, String password, int port) {
        this.host = host;
        this.username = username;
        this.password = password;
        this.port = port;
    }

    public String getHost() {
        return host;
    }

    public String getUsername() {
        return username;
    }

    public String getPassword() {
        return password;
    }

    public int getPort() {
        return port;
    }

}

File: sublib/src/main/java/com/mos/lib/common/http/SubProxy.kt
package com.mos.lib.common.http

class SubProxy(val host: String, val username: String, val password: String, val port: Int) {


    override fun toString(): String {
        return "SubProxy{" +
                "host='" + host + '\''.toString() +
                ", username='" + username + '\''.toString() +
                ", password='" + password + '\''.toString() +
                ", port=" + port +
                '}'.toString()
    }
}
